src/modules/handlers.ts

```diff
@@ -63,14 +72,15 @@ export function onFilesLoaded(): Promise<HTMLElement[]> {
 
         // Some file els can be present on the page but hidden from view when
         // navigating from within the PR view.
-        const readyFileEls = getFileElements().filter(el => {
+        const { core: coreFileEls, unchecked: uncheckedAnnotations } = getFileElements()
+        const readyFileEls = coreFileEls.filter(el => {
           return el.clientHeight > 0 && el.clientWidth > 0
         })
     
         const numFilesChanged = parseInt(filesChangedEl.innerText.trim(), 10)
     
-        if (readyFileEls.length !== numFilesChanged) {
-          Logger.debug('[onFilesLoaded] All files have not loaded yet')
+        if (readyFileEls.length < numFilesChanged) {
```

Comment:
This number was getting jumped which is why in some cases the explorer was never rendering. After this was changed then the rendering failed because of the unexpected structure of the unchecked annotations.